tests/test_celestial.py

```diff
@@ -426,6 +450,34 @@ def test_galactic():
     el,b = anticenter.galactic()
     numpy.testing.assert_almost_equal(el.rad(), pi, decimal=3)
     numpy.testing.assert_almost_equal(b.rad(), 0., decimal=3)
+    t2 = time.time()
+    print 'time for %s = %.2f'%(funcname(),t2-t1)
+
+def test_ecliptic():
+    """Test the conversion from equatorial to ecliptic coordinates."""
+    import time
+    t1 = time.time()
+
+    # Use locations of ecliptic poles from http://en.wikipedia.org/wiki/Ecliptic_pole
+    north_pole = galsim.CelestialCoord(
+        galsim.angle.HMS_Angle('18:00:00.00'),
+        galsim.angle.parse_dms('66:33:38.55')*galsim.degrees)
```

Comment:
The second one here could be written more simply as `galsim.DMS_Angle('66:33:38.55')`.  Similarly, the `angle` part is unnecessary for the RA.

Also, the two other directions to check are the vernal equinox `CelestialCoord(0,0)` and the autumnal equinox `(pi,0)`, which should have the same coordinates in ecliptic coordinates.